doc/RELEASE_STEPS.md

```diff
@@ -14,24 +17,26 @@ Make a release locally:
 - git tag -a -s -m "Libpmemobj-cpp Version $VERSION" $VERSION
 
 Undo temporary release changes:
-- git rm .version
-- git commit -m "common: git versioning"
+- git rm .version && git commit -m "common: git versioning"
```

Comment:
Maybe we should change convention, and have .version file always present in master branch, and increment it in first commit after release.

Reply:
If we decide to change it, I believe it should be done in sep. PR

Reply:
But I believe then we will always have some non-latest version reported by e.g. cmake and pkgconfig file since .version file has priority over git describe, right?

Reply:
if we decide to do this change then we could also change cmake file; but I believe we have working solution and it seems to work just fine with rather low cost of adding/removing `.version` file on each release... 

As a comparison, PMDK repo has approach with VERSION file with release tag and GIT_VERSION file with git formatting pattern. Either of this two is always committed... but, again, I believe our approach is quite all right.